<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrincipalUsuario</class>
 <widget class="QMainWindow" name="PrincipalUsuario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MoreBeaches</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>FormatFactoryAgregar un título.ico</normaloff>FormatFactoryAgregar un título.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(201, 230, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frameTitulo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>881</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
background-color: rgb(0,74,116);
}

QPushButton{
background-color: rgb(0, 170, 255);
border-radius:8px;
}

QPushButton:hover{
background-color: rgb(0, 196, 255);
border-top-left-radius:12px;
border-bottom-left-radius:12px;

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="button_Inicio">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Inicio</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>inicio.png</normaloff>inicio.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="button_Calculo">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string> Calcular</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>pngwing.com(1).png</normaloff>pngwing.com(1).png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="button_Perfil">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Mi Perfil</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>login.png</normaloff>login.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cooper Black</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;MoreBeaches&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>0</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Agregar_un_título-removebg-preview.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>881</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(201, 230, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>10</y>
       <width>901</width>
       <height>561</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(201, 230, 255);</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="pageInicio">
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>871</width>
         <height>211</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>WhitehavenBeach.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>190</y>
         <width>521</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#000000ff;
color: rgb(0, 0, 255);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;Bienvenidos a MoreBeaches&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>275</y>
         <width>401</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>MoreBeaches es una aplicación para el cálculo del Transporte Longitudinal de Arena</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="pageCalculo">
      <widget class="QTableWidget" name="table_Calculos">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>861</width>
         <height>381</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableWidget{
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
gridline-color: rgb(0,74,116);
font-size:12pt;
color: rgb(0, 0, 0);
}
QHeaderView::section{
background-color: rgb(0,74,116);
color: rgb(255, 255, 255);
border:1px solid rgb(0,0,0);
font-size:12pt;

}
QTableWidgetQTableCornerButton::section{
background-color: rgb(255, 255, 255);
border:1px solid rgb(255,255,255);
}
</string>
       </property>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Den_Mar (ρ)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>DensArena (ρs)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Coefic (n)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Altura (Hb)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Angulo (α)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Indice (k)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Resultado</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Fecha</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>361</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #000000ff;
color: rgb(0, 0, 127);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Cálculo Longitudinal de Arena&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Buscar Cálculos</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>90</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Ubicación:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>90</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>90</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(0, 0, 0);</string>
       </property>
       <property name="displayFormat">
        <string>dd/MM/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>90</y>
         <width>47</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Fecha</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_EliminarCalculo">
       <property name="geometry">
        <rect>
         <x>770</x>
         <y>520</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">

QPushButton{
color: rgb(255, 255, 255);
	background-color: rgb(221, 1, 5);
border-radius:8px;
}

QPushButton:hover{
background-color: rgb(255, 120, 129);
border-top-left-radius:12px;
border-bottom-left-radius:12px;

}</string>
       </property>
       <property name="text">
        <string>Eliminar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_NuevoCalculo">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>520</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">

QPushButton{
color: rgb(0, 0, 0);
	background-color: rgb(25, 255, 98);
border-radius:8px;
}

QPushButton:hover{
background-color: rgb(47, 255, 158);
border-top-left-radius:12px;
border-bottom-left-radius:12px;

}</string>
       </property>
       <property name="text">
        <string>Nuevo</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_EditarCalculo">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>520</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(11, 15, 255);
color: rgb(0, 0, 0);
border-radius:8px;
}

QPushButton:hover{
background-color: rgb(16, 132, 255);
border-top-left-radius:12px;
border-bottom-left-radius:12px;

}</string>
       </property>
       <property name="text">
        <string>Editar</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEditHasta">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>90</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(0, 0, 0);</string>
       </property>
       <property name="displayFormat">
        <string>dd/MM/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="Button_Buscar">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>90</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="pageMiPerfil">
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>10</y>
         <width>211</width>
         <height>191</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>login.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_MiPerfil">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>210</y>
         <width>261</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;MI PERFIL&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_Nombre">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>270</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Nombre&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_PApellido">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>270</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Primer Apellido&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_SApellido">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>270</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Segundo Apellido&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>330</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Cambiar Contraseña&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>380</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Contraseña Actual&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditContActual">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>380</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="maxLength">
        <number>20</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>430</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Nueva Contraseña &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditContNueva">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>430</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="maxLength">
        <number>20</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>480</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Confirmar Contraseña &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditConfContr">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>470</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="maxLength">
        <number>20</number>
       </property>
      </widget>
      <widget class="QPushButton" name="button_CerrarS">
       <property name="geometry">
        <rect>
         <x>770</x>
         <y>10</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(255, 68, 35);
border-radius:8px;
}

QPushButton:hover{
background-color: rgb(255, 96, 96);
border-top-left-radius:12px;
border-bottom-left-radius:12px;

}</string>
       </property>
       <property name="text">
        <string>Cerrar Sesión</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>exit-svgrepo-com.png</normaloff>exit-svgrepo-com.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonCambiarContrasena">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>520</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(0, 170, 255);
border-radius:8px;
	color: rgb(0, 0, 0);
}

QPushButton:hover{
background-color: rgb(0, 196, 255);
border-top-left-radius:12px;
border-bottom-left-radius:12px;

}</string>
       </property>
       <property name="text">
        <string>Cambiar</string>
       </property>
      </widget>
      <zorder>label_MiPerfil</zorder>
      <zorder>label_Nombre</zorder>
      <zorder>label_PApellido</zorder>
      <zorder>label_SApellido</zorder>
      <zorder>label_12</zorder>
      <zorder>label_13</zorder>
      <zorder>lineEditContActual</zorder>
      <zorder>label_14</zorder>
      <zorder>lineEditContNueva</zorder>
      <zorder>label_15</zorder>
      <zorder>lineEditConfContr</zorder>
      <zorder>label_6</zorder>
      <zorder>button_CerrarS</zorder>
      <zorder>buttonCambiarContrasena</zorder>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
